app/api/helpers/import_helpers.py

```diff
@@ -355,10 +355,8 @@ def import_event_json(task_handle, zip_path):
         new_event = Event(**data)
         db.session.add(new_event)
         db.session.commit()
-        write_file(
-            path + '/social_links',
-            json.dumps(data.get('social_links', []))
-        )  # save social_links
+        with open(path + '/social_links', 'w') as file:
```

Comment:
why?

Reply:
@CosmicCoder96 i can't use write_file() because it tries to encode the data in utf-8 which gives errors.

Reply:
What errors?

Reply:
@iamareebjamal  `decoding str is not supported`